commit a331150af65477fc3fa72ab341eed5e0b2daf7f3
Author: Joseph Myers <joseph@codesourcery.com>
Date:   Thu Nov 28 20:32:09 2019 +0000

    Update syscall-names.list for Linux 5.4.

    This patch updates syscall-names.list for Linux 5.4.  There are no new
    syscalls, so this is just a matter of updating the version number
    listed in the file.

    Tested with build-many-glibcs.py.

Reworked for (rhbz#1747465)
    Also update copyright info to match upstream.
    Correct some typos in the comments to match upstream.

diff -Nrup a/sysdeps/unix/sysv/linux/syscall-names.list b/sysdeps/unix/sysv/linux/syscall-names.list
--- a/sysdeps/unix/sysv/linux/syscall-names.list        2020-01-24 12:27:15.212564061 -0500
+++ b/sysdeps/unix/sysv/linux/syscall-names.list        2020-01-24 14:42:30.175256876 -0500
@@ -1,5 +1,5 @@
 # List of all known Linux system calls.
-# Copyright (C) 2017-2018 Free Software Foundation, Inc.
+# Copyright (C) 2017-2020 Free Software Foundation, Inc.
 # This file is part of the GNU C Library.
 #
 # The GNU C Library is free software; you can redistribute it and/or
@@ -14,16 +14,15 @@
 #
 # You should have received a copy of the GNU Lesser General Public
 # License along with the GNU C Library; if not, see
-# <http://www.gnu.org/licenses/>.
+# <https://www.gnu.org/licenses/>.

-# This file contains the list of system call names names.  It has to
-# remain in alphabetica order.  Lines which start with # are treated
-# as comments.  This file can list all potential system calls.  The
-# names are only used if the installed kernel headers also provide
-# them.
+# This file contains the list of system call names.  It has to remain in
+# alphabetical order.  Lines which start with # are treated as comments.
+# This file can list all potential system calls.  The names are only
+# used if the installed kernel headers also provide them.

-# The list of system calls is current as of Linux 5.3.
-kernel 5.3
+# The list of system calls is current as of Linux 5.4.
+kernel 5.4

 FAST_atomic_update
 FAST_cmpxchg
